scripts/close-milestone.sh

```diff
@@ -0,0 +1,11 @@
+set -x -e -o pipefail -u
+
+indented_milestones_json_array=$(gh api -H "Accept: application/vnd.github+json" /repos/VirtusLab/git-machete-intellij-plugin/milestones)
```

Comment:
Same concern, is `gh` available in the CI?

Reply:
(see `docker/Dockerfile` + `docker_executor` in CircleCI config)

Reply:
Also... `indented_milestones_...`? 🤔  it's rather `all_milestones_...` or sth like that

Reply:
I noticed that the `gh` api returns the pretty printed json, instead of the compacted one. `jq` seemed to fail in parsing  in that case.  both assigning the result json to a variable, or using `read` would remove the excess whitespace.

Reply:
Hmmm `jq` definitely shouldn't fail depending just on whitespace... could you cook up a repro for that? I.e. a sample such pretty-printed JSON that `jq` would fail on

Reply:
is it a good approach to introduce this?
https://circleci.com/developer/orbs/orb/circleci/github-cli

Reply:
Hmmm in general it'd make sense... although for the sake of consistency I'd rather have `gh` installed in the Docker image, as is the case for all the other software required in the CI.

See CONTRIBUTING.md for the CI image regeneration instructions... the tag should be 8.1.0 according to semver (a non-breaking change). 

Pls provide your DockerHub login (create an account if you don't have one), I'll add you with Write (push) permissions

Reply:
DockerHub login is vzmerr

Reply:
It seems that it is [not a good practice](https://github.com/cli/cli/discussions/3698#discussioncomment-785838) to install `gh` on a Docker container. What do you think?

Reply:
> DockerHub login is vzmerr

Pls check now (https://hub.docker.com/orgs/gitmachete/repositories)

Reply:
> It seems that it is https://github.com/cli/cli/discussions/3698#discussioncomment-785838 to install gh on a Docker container.

I'd say the thread generally speaks in favor of dockerized `gh`, it's quite useful in some cases:
* as per https://github.com/cli/cli/discussions/3698#discussioncomment-3472925, for ease of use in setups like kubernetes
* as per https://github.com/cli/cli/discussions/3698#discussioncomment-3327598, for the use of `gh` in CI pipelines, 

Note that by we do **not** need to use a standalone Docker image with `gh`, it's more of matter of installing `gh` into https://hub.docker.com/repository/docker/gitmachete/intellij-plugin-ci, as `hub` already is (see line 10 in Dockerfile)